pss/pss_test.go

```diff
@@ -47,29 +48,66 @@ func TestTrojanChunkRetrieval(t *testing.T) {
 	payload := []byte("RECOVERY CHUNK")
 	topic := trojan.NewTopic("RECOVERY")
 
-	var ch chunk.Chunk
+	var monitor *Monitor
 
 	pss := NewPss(localStore)
 
 	// call Send to store trojan chunk in localstore
-	if ch, err = pss.Send(ctx, testTargets, topic, payload); err != nil {
+	if monitor, err = pss.Send(ctx, testTargets, topic, payload); err != nil {
 		t.Fatal(err)
 	}
 
 	// verify store, that trojan chunk has been stored correctly
 	var storedChunk chunk.Chunk
-	if storedChunk, err = localStore.Get(ctx, chunk.ModeGetRequest, ch.Address()); err != nil {
+	if storedChunk, err = localStore.Get(ctx, chunk.ModeGetRequest, monitor.chunk.Address()); err != nil {
 		t.Fatal(err)
 	}
 
-	if !reflect.DeepEqual(ch, storedChunk) {
+	if !reflect.DeepEqual(monitor.chunk, storedChunk) {
```

Comment:
i think this might be failing because it looks like the tag id is not persisted in the store.

*if* this is the case, either we:
1. find a way to persist this tag
2. find a different way to compare the chunks

anyway this comparison is a still-unresolved point from the previous review

Reply:
This is my current blocker, there must be another way to compare, or the storage is not correct.

Reply:
see above